### ISSUE TITLE
[Feature request] Option to migrate code for backup or mirroring

### ISSUE THREAD

**Issue by ghost** [vlang/gitly#234]
Hi everyone.

#### concept

I think in addition to adding a p2p storage, having the option to migrate the code to github/gitlab/bitbucket and more platforms. That way we can have an interoperability, backup. So that if one of these platforms stops working, another one won't. 

I use a tool called git-bug which is close to the idea of ​​a protocol like email/matrix. I mean, the issues are distributed on different platforms. I think about integrating the idea of ​​gitbug into gitly, in addition to p2p storage.

In other words, having an option to migrate code for backup or mirroring. Just like, also storage is an interesting idea. There are tools for example non-hosted that work with protocols like remote storage or local storage in the browser like cookies, local storage. These tools are self-hosted, they usually have some p2p protocol as well.

#### question/feedback
I would like to know what the community thinks about this idea.

#### Reference
- https://github.com/vlang/gitly/issues/135
